mcs/class/System.IdentityModel/Makefile

```diff
@@ -7,11 +7,16 @@ OTHER_LIB_MCS_FLAGS = -r:System.Web.ApplicationServices.dll
 endif
 
 LIBRARY = System.IdentityModel.dll
-LIB_REFS = System System.Xml System.Security System.Web System.Configuration Mono.Security System.Runtime.Serialization
+LIB_REFS = System System.Xml System.Security System.Configuration Mono.Security System.Runtime.Serialization
 LIB_MCS_FLAGS = \
 		/d:NET_3_0	\
 	        $(OTHER_LIB_MCS_FLAGS)
 
+XAMMAC_4_5 := $(filter net_4_5_xammac, $(PROFILE))
+ifndef XAMMAC_4_5
```

Comment:
I think a net_4_5_xammac.make like this would do it:

    include $(topdir)/build/profiles/net_4_5.make
    PROFILE_MCS_FLAGS += -d:NO_SYSTEM_DRAWING_DEPENDENCY -d:NO_WINFORMS_DEPENDENCY -d:NO_SYSTEM_WEB_DEPENDENCY -d:XAMMAC_4_5 
    NO_SYSTEM_WEB_DEPENDENCY=1
    NO_WINFORMS_DEPENDENCY=1
    NO_XXX_DEPENDENCY=1


and then in this Makefile, do:

    ifndef NO_SYSTEM_WEB_DEPENDENCY
    LIB_REFS += System.Web
    endif